phpdotnet/phd/Options/Parser.php

```diff
@@ -91,16 +97,15 @@ private function checkOptions() {
     }
 
     public function getopt() {
-        //validate options
-        $this->checkOptions();
+        $this->validateOptions();
 
         $args = getopt($this->getShortOptions(), $this->getLongOptions());
         if ($args === false) {
             trigger_error("Something happend with getopt(), please report a bug", E_USER_ERROR);
         }
 
         foreach ($args as $k => $v) {
-            $handler = $this->handlerForOption($k);
+            $handler = $this->handlerForOption((string) $k);
```

Comment:
Is this cast really needed?

Reply:
Hmm. You're right, strict typing is not used in any of the files and this really is not needed.